spec/versioner/version_file_spec.rb

```diff
@@ -105,12 +135,12 @@
       expect(version_file.current_major_version).to eq '0'
     end
 
-    it 'is at RC iteration 1' do
-      expect(version_file.release_candidate_iteration).to eq '1'
+    it 'is at RC iteration 0' do
+      expect(version_file.release_candidate_iteration).to eq '0'
     end
 
     it 'has RC and the RC version number at the end of the name' do
-      expect(version_file.version).to eq '0.10.0-RC1'
+      expect(version_file.version).to eq '0.10.0-RC.0'
```

Comment:
From https://semver.org/
> Numeric identifiers MUST NOT include leading zeroes.
> Examples: 1.0.0-alpha, 1.0.0-alpha.1

Should we start with `-RC` and continue with `-RC.1`?

Reply:
hmmmm... I was going off of the semver npm package: https://www.npmjs.com/package/semver which is the package I'm going to be using to automate release management.

If you run `npx semver 3.99.0 -i preminor --preid RC` from the command line it will give back `3.99.0-RC.0`

Reply:
I am as well. The argument is very reasonable, a lonely 0 is not a leading zero.